apps/pg_subscriber/lib/pg_subscriber/messages/update.ex

```diff
@@ -0,0 +1,41 @@
+defmodule PgSubscriber.Messages.Update do
```

Comment:
Like this. Was thinking the same. If we create interface and one impl for each message type we will not even need `handle_...` functions. We can just do
```elixir
defp handle_message(message) do
  <<msg_type::utf8, msg::binary>> = message
  handler = case <<msg_type>> do
     "B" -> Begin
     "U" -> Update
      ...
  end

  Logger.info("Got {handler.type} msg")
  message = handler.fromData!(msg)
  Logger.info(message: message)
  message
end
```